atlp-rwanda/alpha-ec-be
# Feat(Fake Ads):Ads Management

### issue in referencing?

### Questions:

#### What does this PR do?

Fake Ads management

#### Description of Task to be completed?

Add a docker service in our docker network to host a node.js middleware . this will host a serveless framework that runs a AWS lamda function / functions to perform ads management


#### How should this be manually tested?

- download and install `Docker Desktop`
- run `docker-compose build`
- run `docker-compose up -d`
- check the [swagger documentation ](http://localhost:3000/swagger/)

#### Any background context you want to provide?

-dummy data are provided before having the real data

#### What are the relevant pivotal tracker/Trello stories?

- id of the project #187535721

#### Questions:
